cumulusci/core/flowrunner.py

```diff
@@ -337,9 +328,6 @@ def _visit_step(
         """
         number = LooseVersion(str(number))
 
-        if visited_steps is None:
```

Comment:
i'd kinda prefer to leave this in and remove the explicit passed `[]` in _init_steps, it feels like the intent is clearer, and we'd still get coverage. but fine.

Reply:
Sure, I don't feel strongly about it.